xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -36,6 +36,7 @@
 #include "settings/lib/Setting.h"
 #include "settings/lib/SettingDefinitions.h"
 #include "storage/MediaManager.h"
+#include "utils/log.h"
```

Comment:
move it two lines down

> Included header files have to be sorted (case sensitive) alphabetically [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
Fixed.